### DISCUSSION TITLE
Welcome to op Discussions!

### DISCUSSION THREAD

**Discussion by WallBreaker2** [WallBreaker2/op#78] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by LUOjq002** [comment]
项目里的例程太老旧了，并且按照项目的配置方法打印不出版本号

**Comment by oltramarino** [comment]
群满了，有考虑建一个新的吗，或者建个tg群之类的

**Reply by WallBreaker2** [reply]
请加大群27872381

**Reply by chang-chih-yao** [reply]
有微信或TG群嗎? 海外用戶進不去🥲
